docker/nginx/docker-entrypoint.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -eo pipefail
+
+if [ "$1" = 'nginx' ]; then
```

Comment:
slightly different then the one I had for uwsgi?

if [ "$1" = 'uwsgi' -a "$(id -u)" = '0' ]; then


Do we care?